### ISSUE TITLE
[EN][Quest][The Storm Peaks] The Brothers Bronzebeard

### ISSUE THREAD

**Issue by podatekvat** [Atlantiss/BugTracker#9380]
[//]: # (***************************************************************)
[//]: # (** DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED **)
[//]: # (***************************************************************)

[//]: # (On the Bugtracker all titles of reports must be written in English so that everyone, no matter which language they use everyday, could search and check if the problem already has been reported by another player. When describing the problem you can use English as well as Polish and in nearest future - others. It is important to determine at the beginning of a title in which language it is written by using symbols [EN] or [PL].)

[//]: # (Enclose links to things related to the bug using http://wowhead.com or http://cata.openwow.com)
[//]: # (Write your tickets according to the format:)
[//]: # ([EN][Quest][Zone] A Vision of the Past)
[//]: # ([EN][NPC] Thoralius the Wise)
[//]: # ([EN][Spell][Class] Frostfire Bolt)
[//]: # ([PL][Talent][Class] Brain Freeze)
[//]: # ([EN][Glyph][Class] Glyph of Frostfire Bolt)
[//]: # ([EN][Npc][Drop] Thoralius the Wise)
[//]: # ([EN][Web] Armory doesnt work)

**Description:** http://www.wowhead.com/quest=12973/the-brothers-bronzebeard This bug is very nasty and once it "infect" your character then you're royaly screwed. The quest works like it's intended. Full blizzlike. But it leaves u with a phase in the center of Storm Peaks - Engine of the makers, which should disappear after u finish the quest. Phase description: whole area is crowded with Iron Dwarves. Description how correct phase after compliting the quest should looks like: the area is crowded with rhinos, frost wyrms and there is a patrolling quest npc Creteus. This makes you unable to finish following quests:
http://www.wowhead.com/quest=12877/the-lonesome-watcher - no proper npc Creteus
http://www.wowhead.com/quest=12865/loyal-companions - no rhinos to kill
http://www.wowhead.com/quest=12855/sniffing-out-the-perpetrator - no quest npc to kill
It also makes you unable to see Time-lost Protodrake, Dirkee and Vyragosa in the area which makes you really sad if u want to do Frostbitten achievement www.wowhead.com/achievement=2257/frostbitten

**Current behaviour:** When you finish quest The Brothers Bronzebeard it will leave you with a phased location of Engine of the Makers.

**Expected behaviour:** When you finish the quest then Engine of the Makers location should go back to normal phase (with rhinos and wyrms, etc)

**Steps to reproduce the problem:**

1. First of all take a look how the area looks like in the current phase, then take the quest The Brothers Bronzebeard
2. Look how phase changed in the area while on the quest
3. Finish the quest
4. Go back to the engine of the makers area and look how the phase didnt change to the previous one = you are screwed

rev 5464 hash

**Comment by Piklo** [comment]
Why did you make a new post exactly the same as the previous one? #9299

**Comment by AngryEndUser** [comment]
+1 . quest raped more than one of my toons.

**Comment by podatekvat** [comment]
I closed previous one and reposted it couse noone from the support reacted to the issue which - I assume from my experience - it probably wont be taken care of. It's a major problem for many players so I'm doing what I can to point it out.

**Comment by emilius** [comment]
+1. By doing this quest on accident, now Northland Lormaster can't be completed. GM can remove the phasing but it only works for as long you can stay in that area, afterwards you get it back. This is quite annoying since the phasing applies to multiple parts of Storm Peaks.
The easiest thing to do is to remove the phasing all together. The player will have a worse visual experience  for that quest but questing in the zone will not become a nightmare.

**Comment by tomatosz** [comment]
Does anyone have proof that the phasing should go away when you finish the quest? I can't find any useful info online, and as i see it now there are two options:
1. phasing issue; the phasing should go away, as you suggested
2. quest order issue; the quests that are incompletable should be completed BEFORE this quest (if this is the case, the rares should have their phasing updated)

**Comment by podatekvat** [comment]
Yep, it's quests order issue. You can do http://www.wowhead.com/quest=12973/the-brothers-bronzebeard BEFORE you get access to http://www.wowhead.com/quest=12877/the-lonesome-watcher which cannot be completed in changed phase. You suppose to do whole chain quest (starting with http://www.wowhead.com/quest=12871/aid-from-the-explorers-league) and then "Brothers Bronzebeard" quest should unlock at the very end of chain. But the second problem is, a whole lot of people already did that quest and are stuck in the wrong phase.
